test/test_activity_update.py

```diff
@@ -27,7 +27,6 @@ def test_user_creation(self):
         user = UserFactory.create()
         update_feed = get_update_feed()
         err_msg = "It should be the same user"
-        assert update_feed[0]['id'] == user.id, err_msg
```

Comment:
ok, I only wanted to know the reason for removing it. I agree it's better this way (removing it then).